lib/matplotlib/backend_bases.py

```diff
@@ -3224,7 +3226,26 @@ def on_tool_fig_close(e):
         return self.subplot_tool
 
     def save_figure(self, *args):
-        """Save the current figure."""
+        """
+        Save the current figure.
+
+        Backend implementations may choose to return
+        the absolute path of the saved file, if any, as
+        a string.
+
+        If no file is created then `None` is returned.
+
+        If the backend does not implement this functionality
+        then `NavigationToolbar2.UNKNOWN_SAVED_STATUS` is returned.
```

Comment:
Repeating my self a bit from the main comments, the problem is that backends that don't implement _any_ of this still return `None`.